src/index.html

```diff
@@ -2,12 +2,53 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport"  content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card-block" data-qa="card">
+      <div class="product-description">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="product-description__photo"
+        >
+        <h2 class="product-description__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="product-description__code">Product code: 195434</p>
+      </div>
+
+      <div class="review">
+        <section class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </section>
+        <p class="review__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="price">
+        <p class="price__text">Price:</p>
+        <h1 class="price__number">$2,199</h1>
+      </div>
+
+      <div class="buy">
```

Comment:
we make element a BEM block if we can reuse it somewhere, as an independent block

it seems to me that we will not be able to reuse these elements anywhere, so it is better to make them simple elements of the card block

you must to have only two blocks here: "card" and "stars"

Reply:
Rename class according to the BEM